benchmarks/osperf/Kconfig

```diff
@@ -6,6 +6,8 @@
 config BENCHMARK_OSPERF
 	tristate "System performance profiling"
 	default n
+	select PIPES
+	select SCHED_HPWORK
```

Comment:
it's better to change the `select` to `depend on`

Reply:
Hi @xiaoxiang781216, thanks for the reply. If you don't mind, for my better understanding, can you provide me with the reason why it's better to use `depend on`?

Reply:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-recommendations

Reply:
@xiaoxiang781216, thanks for the document. I've made the necessary changes and now use the "depends on" directives. I'd appreciate it if you could take the time to review the changes and merge them upstream if they are satisfactory.

Reply:
Sure, thanks.